frontend/src/pages/details-page.tsx

```diff
@@ -8,36 +8,38 @@ const DetailsPage = () => {
   const post: Post = state?.post;
   const navigate = useNavigate();
   return (
-    <div className="w-full font-[Poppins] ">
-      <div className="relative flex flex-col">
-        <img
-          src={post.imageLink}
-          alt={post.title}
-          className="h-[360px] md:h-[460px] object-cover w-full"
-        />
+    <div className="dark:dark w-full min-h-screen">
+      <div className="w-full font-[Poppins]">
```

Comment:
Okay if you have used dark:dark here why not all the places the same? some has dark:text-white this one has dark:dark, i dont see consistency and the need of this extra confusing custom dark utility class. kindly confirm once

Reply:
sir some places we only need text as black. and some places we need background and text both as black

Reply:
what is the case where both background and text are black ? I didn't get you, and dark:dark only for this place I don't think it is efficient you can directly replace class names inside the dark here, it is not good dev experience

Reply:
ok sir I remove it